src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,36 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+	// iOS localization specific options
+    // TODO: refactor the code for DRY issues
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    // using NSNull comparison for JSON values set to null
+	if(command.arguments.count == 3 && command.arguments[2]) {
+		NSDictionary *feedbackOptions = command.arguments[2];
+
+		NSDictionary *iOSLocalizationOptions = [feedbackOptions valueForKey:iOSFeedbackOptionsKeyName];
+		if(iOSLocalizationOptions != nil ) {
+			NSString* feedbackSentAlertTitle = [iOSLocalizationOptions valueForKey:feedbackSentAlertTitleSettingName];
+			if (feedbackSentAlertTitle != nil && ![feedbackSentAlertTitle isKindOfClass:[NSNull class]]) {	
```

Comment:
Let's keep this with square brackets to keep it consistent with the other part of the implementation.